### PR TITLE
fix mappers

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by BelehovEgor — APPROVED** [review]

File: src/CompanyService.Mappers/Db/DbPositionMapper.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public DbPosition Map(CreatePositionRequest value, Guid companyId)
                 Id = Guid.NewGuid(),
                 CompanyId = companyId,
                 Name = value.Name,
-                Description = value.Description.Any() ? value.Description : null,
+                Description = value.Description != null && value.Description.Any() ? value.Description : null,
```
Comment (TIove) [inline src/CompanyService.Mappers/Db/DbPositionMapper.cs:24]: value.Description.Trim().Any()  мб?

**Review by djino97 — APPROVED** [review]

File: src/CompanyService.Mappers/Db/DbPositionMapper.cs (hunk 1 above)
Comment (borovHokage) [reply src/CompanyService.Mappers/Db/DbPositionMapper.cs]: and may be  ? value.Description.Trim() : null

File: src/CompanyService.Mappers/Db/DbPositionMapper.cs [hunk 2]
```diff
@@ -21,7 +21,7 @@ public DbPosition Map(CreatePositionRequest value, Guid companyId)
                 Id = Guid.NewGuid(),
                 CompanyId = companyId,
                 Name = value.Name,
-                Description = value.Description.Any() ? value.Description : null,
+                Description = value.Description != null && value.Description.Trim().Any() ? value.Description : null,
```
Comment (TIove) [inline src/CompanyService.Mappers/Db/DbPositionMapper.cs:24]: value.Description.Trim()

File: src/CompanyService.Mappers/Db/DbPositionMapper.cs (hunk 1 above)
Comment (MariyaSin) [reply src/CompanyService.Mappers/Db/DbPositionMapper.cs]: it does not work, bool not nullable

Comment (MariyaSin) [reply src/CompanyService.Mappers/Db/DbPositionMapper.cs]: fixed

File: src/CompanyService.Mappers/Db/DbPositionMapper.cs (hunk 2 above)
Comment (TIove) [reply src/CompanyService.Mappers/Db/DbPositionMapper.cs:24]: It should be like
 Description = value.Description != null && value.Description.Trim().Any() ? value.Description.Trim() : null,

Comment (MariyaSin) [reply src/CompanyService.Mappers/Db/DbPositionMapper.cs]: done

**Review by TIove — APPROVED** [review]

**Pull request merged**